app/main.py

```diff
@@ -1,8 +1,25 @@
 class Person:
-    # write your code here
-    pass
+
+    people = {}
+
+    def __init__(self, name: int, age: int) -> None:
```

Comment:
The `name` parameter in the `__init__` method should be of type `str`, not `int`, as names are typically strings.